postr/apis/discord.py

```diff
@@ -1,107 +0,0 @@
-# api interface that all api classes must extend
```

Comment:
This wasn't actually deleted, it got moved but github thinks it was deleted because there were so many changes to it. Scroll down.